TaintTracker/TaintTracker.cpp

```diff
@@ -4,19 +4,36 @@
 #include <fstream>
 #include <iostream>
 #include <list>
+/*
+ * BASE VARIABLE TO CHECK LIBC
+ */
+#define LIBC_BASE 0x70000000000
+
+/* BASIC TAINT AREA STRUCTURE */
+struct range
+{
+  UINT64 start;
+  UINT64 end;
+};
+
+std::list<struct range> bytesTainted;
 
 /*
  * BASIC UTILITY FUNCTIONS
  */
 int usage()
 {
-    cerr << "Not Properly Used" << endl;
+    cerr << "Usage: ./TaintTracker file" << endl;
     return -1;
 }
 
-bool isLibraryFunction() {
-    // TODO : Figure out to do this
-    return true;
+bool isLibraryFunction( UINT64 startAddr) {
+    if (startAddr >= LIBC_BASE ){
+      return true;
+    }
+    else {
+      return false;
+    }
```

Comment:
I think @Freakston or @adityavardhanpadala has a better solution. Please discuss in the issue

Reply:
Okay I will discuss with them